extensions/amp-ad-exit/0.1/config.js

```diff
@@ -121,6 +134,21 @@ function assertTarget(name, target, config) {
       user().assert(
           pattern.test(variable), '\'%s\' must match the pattern \'%s\'',
           variable, pattern);
+      const vendor = target.vars[variable]['vendorAnalyticsSource'];
+      if (vendor !== undefined) {
+        if (getMode().test) {
```

Comment:
I think different messages for test vs non-test mode is unnecessary.  Instead just have the two user asserts (dev would see user error)